lib/uncruft/railtie.rb

```diff
@@ -11,12 +11,16 @@ class Railtie < ::Rails::Railtie
         strategies.unshift(DeprecationHandler.new)
         config.active_support.deprecation = strategies
       end
+    end
 
-      if Rails.gem_version >= Gem::Version.new('7.1')
-        initializer "uncruft.deprecator" do |app|
-          app.deprecators[:uncruft] = Uncruft.deprecator
-        end
+    if Rails.gem_version >= Gem::Version.new('7.1')
+      initializer "uncruft.deprecator" do |app|
+        app.deprecators[:uncruft] = Uncruft.deprecator
       end
     end
```

Comment:
This was only running in development and test, which is wrong. It should always run.